packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -421,6 +421,7 @@ function useMediaUploadQueue() {
       finishUploading({
         id,
         resource: newResource,
+        additionalData
```

Comment:
This (and the change to the reducer) seem a bit pointless. It just passes through `additionalData` without modifying it.

Reply:
Will re-check ... at one point I wasn't seeing the data I added coming in.

Reply:
Seems fine without 

https://github.com/GoogleForCreators/web-stories-wp/pull/12335/commits/b3531bc0f5607b9ca26cfea30cf4631ed137a124

https://github.com/GoogleForCreators/web-stories-wp/pull/12335/commits/b2ea3a589d676309ed6e5821dec23423552a8af2